website/client/components/static/header.vue

```diff
@@ -35,13 +35,20 @@
 <style lang='scss' scoped>
   @import '~client/assets/scss/colors.scss';
 
+  @import url('https://fonts.googleapis.com/css?family=Varela+Round');
```

Comment:
In a previous comment you said

> It's scoped css, so normally shouldn't propagate outside of those static pages. But seems I can just declare the import on the home static page, the static header seems to catch up on it. It's not loading in the app for sure, I think the scoped css is making sure of it, and I checked as well with the dev toolbar if the Valera font was loaded in the app, and it's not. So should be safe on that part.

so maybe this second import can be removed?

Reply:
Ok let me check that!